test/txsim/run_test.go

```diff
@@ -151,6 +152,8 @@ func Setup(t testing.TB) (keyring.Keyring, string, string) {
 	t.Helper()
 
 	cfg := testnode.DefaultConfig().WithTimeoutCommit(300 * time.Millisecond).WithFundedAccounts("txsim-master")
+	// Use priority mempool for consistent simulator behavior
```

Comment:
[nit] is simulator == txsim?

```suggestion
	// Use priority mempool for consistent txsim behavior
```